### DISCUSSION TITLE
Welcome to Apigee_to_APIM_migration_tool Discussions!

### DISCUSSION THREAD

**Discussion by pfekrati** [Azure/Apigee_to_APIM_migration_tool#13] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by BrunoLopes** [comment]
Hello, I work as a solution architect at Avanade and I would like to know if there is an average success rate for policies migrations and if you could share any information from a real world migration case.
